src/RepoUtil/repoutil.runtimeconfig.json

```diff
@@ -0,0 +1,8 @@
+{
+  "runtimeOptions": {
+    "framework": {
+      "name": "Microsoft.NETCore.App",
+      "version": "1.0.0-rc3-002733"
```

Comment:
Why is the version of Microsoft.NetCore.App different here as compared to project.json?